posthog/migrations/0299_recording_opt_in_default_true.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.16 on 2023-01-31 17:09
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
We will need to ensure that this migration is also followed up with by the new logical replica. cc @guidoiaquinti @hazzadous

Reply:
No need, we're gonna nix the migration 👍